### PR TITLE
fix: correct keystone_domains reference from horizon

### PR DESCRIPTION
**Pull request by ricolin**
fix https://github.com/vexxhost/atmosphere/issues/727

### CODE REVIEW

**Comment by ricolin** [comment]
recheck

File: roles/horizon/vars/main.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ _horizon_helm_values:
             sso:
               enabled: true
               initial_choice: "{{ (keystone_domains is defined) | ternary(keystone_domains[0].name, 'atmosphere') }}"
-            idp_mapping: "{{ keystone_domains | default([{'name': 'atmosphere', 'label': 'Atmosphere'}]) | vexxhost.atmosphere.keystone_domains_to_idp_mappings }}" # noqa: yaml[line-length]
```
Comment (mnaser) [inline roles/horizon/vars/main.yml:34]: @ricolin I think the cleaner way to do this with making less changes (but it's very stinky).

```yaml
idp_mapping: "{{ keystone_domains | default([{'name': (keystone_keycloak_realm | default('atmosphere')), 'label': (keystone_keycloak_realm_name | default('Atmosphere'))}]) | vexxhost.atmosphere.keystone_domains_to_idp_mappings }}" # noqa: yaml[line-length]
```

**Review by mnaser — CHANGES_REQUESTED** [review]

**Review by mnaser — APPROVED** [review]

**Pull request merged**